### ISSUE TITLE
It's too easy to miss a timer-ready-event

### ISSUE THREAD

**Issue by lual** [Zren/plasma-applet-eventcalendar#296]
If timer time is up - please make the user to affirm it, by clicking "Dismiss/Acknowledged" somewhere.
Make tray info red, if time is up and not dismissed.
Show overdue time. ("Overdue: * Minutes." )
Bonus-wish: Display running timer as monocolored "progress bar" in "System Tray".

By the way: I've tried also your plasma-applet-systraytimer. This works much better as timer, because of the status info in the tray. But lacks also a "Time is up - acknowledged"-button.

And anyway - thanks for this useful applet.

**Comment by santanoce** [comment]
> If timer time is up - please make the user to affirm it, by clicking "Dismiss/Acknowledged" somewhere.

I agree that it's too easy to miss a timer-ready-event, and I solved this issue by modifying the file "~/.local/share/plasma/plasmoids/org.kde.plasma.eventcalendar/contents/ui/TimerModel.qml on line 150:
`expireTimeout: 0`
This way the notification will persist until the user has manually dismissed it.
